src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +58,41 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
-                        fastAndFurious.getId(), LocalDate.now()));
+                fastAndFurious.getId(), LocalDate.now()));
+
+        UserService userService = (UserService)
+                injector.getInstance(UserService.class);
+        User user = new User();
+        user.setEmail("bob@gmail.com");
+        user.setPassword("12345");
+        userService.add(user);
+
+        TicketDao ticketDao = (TicketDao)
+                injector.getInstance(TicketDao.class);
```

Comment:
Directly using `TicketDao` instead of a `TicketService` might not align with the intended service-oriented architecture. Consider using a service to handle ticket-related operations.

Reply:
There isn't a service layer like TicketService in my application.